src/app/apps/lti/forms.py

```diff
@@ -0,0 +1,37 @@
+"""Warren LTI app forms"""
+
+from django import forms
+from django.core import signing
+from django.core.exceptions import ValidationError
+from django.utils.translation import gettext_lazy as _
+
+
+class BaseLTIUserForm(forms.Form):
+    """Warren LTI user form.
+
+    This form is used to validate user initially authenticated via LTI.
+    """
+
+    platform = forms.URLField(required=True)
+    course = forms.CharField(required=True, max_length=100)
+    user = forms.CharField(required=True, max_length=100)
+    email = forms.EmailField(required=True)
+
+
+class SignedLTIUserForm(forms.Form):
+    """A LTIUserForm with a valid signature."""
```

Comment:
Nit: to be consistent with l.10
```suggestion
    """A LTI user form with a valid signature."""
```